Sources/PT.PM.PythonParseTreeUst/PythonAntlrConverter.cs

```diff
@@ -44,6 +44,14 @@ public Ust VisitFile_input(PythonParser.File_inputContext context)
             {
                 block.Statements.Add(Visit(child).ToStatementIfRequired());
             }
+
+            if (root.CurrentSourceFile?.Name.Length > 0)
+            {
+                var dotIndex = root.CurrentSourceFile.Name.LastIndexOf('.');
+                return new NamespaceDeclaration(new StringLiteral(root.CurrentSourceFile?.Name
```

Comment:
Could you please reformat this like
```cs
return new NamespaceDeclaration(
    new StringLiteral(root.CurrentSourceFile?.Name.Remove(dotIndex < 0 ? 0 : dotIndex)),
    block,
    context.GetTextSpan());
```
? Otherwise it's hard to follow where one argument ends and the next one starts.